### PR TITLE
Clean bokeh code

### PR DESCRIPTION
**Pull request by ahartikainen**
Docstring and some simple changes.

### CODE REVIEW

File: arviz/plots/backends/bokeh/__init__.py [hunk 1]
```diff
@@ -30,5 +31,22 @@ def copy_docstring(lib, function):
     return doc
 
 
+def check_bokeh_version():
```
Comment (canyon289) [inline arviz/plots/backends/bokeh/__init__.py]: In new PR we can add this into the new method added if that sounds good

File: arviz/plots/backends/bokeh/bokeh_parallelplot.py [hunk 2]
```diff
@@ -3,26 +3,18 @@
...
-                "poly_select",
-                "undo",
-                "redo",
-                "reset",
-                "save,hover",
-            ]
-        )
+        tools = rcParams["plot.bokeh.tools"]
+        output_backend = rcParams["plot.bokeh.output_backend"]
+        dpi = rcParams["plot.bokeh.figure.dpi"]
```
Comment (canyon289) [inline arviz/plots/backends/bokeh/bokeh_parallelplot.py]: This pattern means if people want to change things they need to both modify RcParams or arg, so api is sort of split. Is my understanding correct?

**Comment by canyon289** [comment]
Scanning over this looks good to me!

File: arviz/plots/backends/bokeh/bokeh_parallelplot.py (hunk 2 above)
Comment (ahartikainen) [reply arviz/plots/backends/bokeh/bokeh_parallelplot.py]: Api is split now, until we fix the api

File: arviz/plots/backends/bokeh/__init__.py (hunk 1 above)
Comment (ahartikainen) [reply arviz/plots/backends/bokeh/__init__.py]: Sounds ok

File: arviz/plots/backends/bokeh/bokeh_parallelplot.py (hunk 2 above)
Comment (canyon289) [reply arviz/plots/backends/bokeh/bokeh_parallelplot.py]: Works for me

**Pull request merged**